src/test/java/org/takes/rs/RsWithTypeTest.java

```diff
@@ -151,22 +174,38 @@ public void replacesTypeWithHtml() throws Exception {
 
     /**
      * RsWithType.JSON can replace an existing type with application/json.
-     * @throws java.lang.Exception If a problem occurs.
+     * @throws Exception If a problem occurs.
      */
     @Test
     public void replacesTypeWithJson() throws Exception {
         MatcherAssert.assertThat(
             new RsPrint(
+                new RsWithType.JSON(new RsWithType(new RsEmpty(), TYPE_XML))
```

Comment:
Because the goal of this unit test (that was not created by me) is to show that we can replace the previous content type which is text/xml here with a new content type which is application/json in that case.